raiden/tests/integration/api/test_restapi.py

```diff
@@ -1797,7 +1798,8 @@ def test_pending_transfers_endpoint(raiden_network, token_addresses):
     mediator_server = create_api_server(mediator, 8576)
     target_server = create_api_server(target, 8577)
 
-    target.raiden.raiden_event_handler = target_hold = HoldRaidenEvent()
+    raiden_event_handler = RaidenEventHandler()
```

Comment:
Hey, do you mind removing these? A few commits ago the `raiden_network` and `raiden_chain` fixtures started instantiating the apps with the `HoldRaidenEventHandler`, so we don't need to monkey patch it. (It is done [here](https://github.com/raiden-network/raiden/pull/4052/files#diff-1ad623a4294804eb3129dc803013bdddR376))